utils/frame/remote-externalities/src/lib.rs

```diff
@@ -491,6 +418,12 @@ impl<B: BlockT> Builder<B> {
 		self
 	}
 
+	/// Inject a manual raw prefixes
```

Comment:
dq: raw_prefix means hash_prefix? am I supposed to call `raw_prefix(&twox_128(bytes))` for this to work?

Reply:
>  Am I supposed to call raw_prefix(&twox_128(bytes)) for this to work?

Yeah, combined with your above feedback, maybe I should call it `inject_hashed_prfix`. 

By raw, I wanted to mean that we don't do anything with it, but that is actually the exact opposite of what raw means :D